server/api/controllers/member.js/add-member.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  friendlyName: 'Add members',
+  description: '',
+  inputs: {
+    member: {
```

Comment:
are you trying to create a controller that receives a member type object and saves it? if so, can just make this a json type

```
inputs: {
  member: {
    type: 'json',
    required: true,
    description: 'member object'
  }
}
```

Reply:
ohh yea this is what im trying to do. this is helpful